nixos/modules/config/xdg/icons.nix

```diff
@@ -19,9 +19,15 @@ with lib;
       "/share/pixmaps"
     ];
 
-    environment.profileRelativeSessionVariables = {
-      XCURSOR_PATH = [ "/share/icons" ];
-    };
+    environment.profileRelativeSessionVariables.XCURSOR_PATH = [
+      "/share/icons"
+      "/share/pixmaps"
```

Comment:
I don't suppose we can make it explicit that these are in terms of `XDG_DATA_DIRS` somehow?

Reply:
`/$XDG_DATA_DIRS/icons`?

Reply:
That won't work, will it? Maybe we could do something if this was bash, but given that it's the more restrictive PAM format maybe not.

Reply:
Yeah, that is not possible. `XDG_DATA_DIRS` might be even incomplete at the time it is set. The patch for libxcursor is the only sane way.

Reply:
Correct, I don't think there's a way we can do this.